K2Bridge.Tests.UnitTests/KustoDAL/CslQueryProviderExtensionsTests.cs

```diff
@@ -59,7 +59,7 @@ public void ExecuteMonitoredQuery_WithNullQuery_ThrowsException()
         {
             Assert.ThrowsAsync(
                 Is.TypeOf<ArgumentNullException>()
-                 .And.Message.EqualTo("query cannot be null (Parameter 'query')"),
+                .And.Message.EqualTo("query cannot be null (Parameter 'query')"),
```

Comment:
Shouldn't this (and similar) be indented?

Reply:
This is consistent with the VS format command. Did you mean it should start in the next tab?

Reply:
yes